libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/NavigationEngineFactoryTest.java

```diff
@@ -63,10 +63,14 @@ public void updateCameraEngine_ignoresNull() {
 
   @Test
   public void updateSnapEngine_ignoresNull() {
-    NavigationEngineFactory provider = new NavigationEngineFactory();
+    NavigationEngineFactory provider = buildNavigationEngineFactory();
 
     provider.updateSnapEngine(null);
 
     assertNotNull(provider.retrieveSnapEngine());
   }
+
+  private NavigationEngineFactory buildNavigationEngineFactory() {
+    return new NavigationEngineFactory();
```

Comment:
Does this really add any value? If you want to initialize with some default data you can do a faker like `BannerComponentsFaker`.